README.rst

```diff
@@ -27,16 +27,285 @@
 
 |License|
 
-NVIDIA TensorRT Inference Server
-================================
+NVIDIA Triton Inference Server
+==============================
 
-    **NOTE: You are currently on the r20.03 branch which tracks
```

Comment:
Include note about the new name.

Reply:
You want a note at the top here?

Reply:
Look at the master README, there is bold text that starts with "NEW NAME", we want that included in this README as well.

Reply:
Hold, let me fix that then